deploy-git-hook

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+
+if [ -f .git ];
+then
+	LOCATION="$(cat .git | awk '{print $2}')"
```

Comment:
instead of 'cat file | awk', you can read the file directly with awk by:

awk '{print $2}' < .git

This would be the more "bash-y" way of doing this.